todo/src/containers/container.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+import { addTodo, removeTodo, toggleTodo } from '../actions/todoActions';
+
+class Todo extends Component {
+    state = {
+        todos: '',
+        toggle: false
+    }
+
+updateTodo = (event) => {
+    console.log(event.target);
+    this.setState({todos: event.target.value});
+}
+
+addTodoInput = (event) => {
+   const {todos, toggle} = this.state
+    console.log(event.target);
+    event.preventDefault();
+    this.props.addTodo({todos, toggle});
+}
+
+removeTodoInput = (event) => {
+    this.props.removeTodo(event.target.getAttribute('id'));
+}
+
+toggleTodoInput = (event) => {
+    this.props.toggleTodo(event.target.getAttribute('id'));
+}
+
+render() {
+    console.log(this.props);
+    return (
+        <div className="Todo">
+            <form onSubmit={this.addTodoInput}>
+            <input
+                onChange={this.updateTodo}
+                placeholder="todo"
+                value={this.state.todos}
+            />
+            <button value="addTodo" onClick={this.addTodoInput} type="submit">Add to todolist</button>
+            </form>
+            <ul>
+                {this.props.todos.map((todo, index) => {
+                    console.log(todo.toggle);
+                    return(
```

Comment:
So you can actually just do this to add `removeTodo` and `toggleTodo` functionality:

```
<React.Fragment key={index}>
    <li id={todo.id} onClick={this.toggleTodoInput} className={todo.toggle ? 'line-through' : null}>{todo.todos}</li>
    <button id={todo.id} onClick={this.removeTodoInput} />
</React.Fragment>
```

Reply:
This just came to me too, but you can call `this.props.toggleTodo` and pass it `todo.id` instead of calling `toggleTodoInput` and `removeTodoInput`.